examples/80_extending/example_extending_data_preprocessor.py

```diff
@@ -82,6 +79,9 @@ def get_hyperparameter_search_space(dataset_properties=None):
 )
 clf.fit(X_train, y_train)
 
+assert len(clf.get_models_with_weights()) > 0  # Just to make sure models were found
```

Comment:
Could you please extend the comment so that it is more standalone?

Reply:
Done